lib/internal/util/inspect.js

```diff
@@ -861,8 +862,32 @@ function formatPrimitive(fn, value, ctx) {
   return fn(value.toString(), 'symbol');
 }
 
-function formatError(value) {
-  return value.stack || errorToString(value);
+function formatError(value, constructorName) {
+
+  if (value.stack) {
+    let stack = value.stack;
+    if (!stack.startsWith(constructorName) &&
+        !(stack.startsWith('file:') && !constructorName.startsWith('file'))) {
```

Comment:
Could you add comments that explain these checks? I’m not sure I quite understand what situations they refer to…

Reply:
@antsmartian the part before the `SyntaxError` is not part of the stack trace. This is extra information returned to the user when the application closes due to the exception and is not special about ecmascript modules. Similar things will be printed with all errors.

Reply:
Oh my bad. I guess I got it confused. Then I will remove it.